src/components/Table/CollapseToggle.test.tsx

```diff
@@ -1,11 +1,15 @@
 import { CollapseToggle } from "src/components/Table/CollapseToggle";
-import { GridCollapseContext } from "src/components/Table/GridTable";
-import { noop } from "src/utils";
+import { RowState, RowStateContext } from "src/components/Table/RowState";
 import { render } from "src/utils/rtl";
 
 describe("CollapseToggle", () => {
   it("defaults to uncollapsed", async () => {
-    const r = await render(<CollapseToggle row={{ id: "r:1", kind: "header" }} />);
+    const rowState = new RowState(null!, undefined);
```

Comment:
Just updating these tests to get dummy `RowStateContext`s instead of the prior dummy `GridCollapseContext`